CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 [//]: # (current developments)
 
+## 2023-01-17   3.4:
```

Comment:
```suggestion
## 2023-01-17   3.4.0:
```

It looks like previous released did include the `.0` trailer (see 3.3.0).

Reply:
done